<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="dell-PC" ignored="0" name="Test1" tests="1" failures="0" timestamp="20 Aug 2017 17:32:23 GMT" time="220.729" errors="0">
  <testcase name="@AfterTest afterClass" time="5.001" classname="offline_Testing_Pro.Offline_Web_App">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at offline_Testing_Pro.Offline_Web_App.afterClass(Offline_Web_App.java:110)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- @AfterTest afterClass -->
  <testcase name="f" time="48.257" classname="offline_Testing_Pro.Offline_Web_App"/>
</testsuite> <!-- Test1 -->
